Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,167 @@
+
+                int convertedIntAge = ConvertToInt("\nEnter your age, no '.'");
+                Console.WriteLine($"Your converted int is \"{convertedIntAge}\"");
+
+                float convertedFloatHeight = ConvertToFloat("\nEnter your height in meters. '.' is ok to use.");
+                Console.WriteLine($"Your converted float is \"{convertedFloatHeight}\"");
+
+                float convertedFloatWeight = ConvertToFloat("\nEnter your weight in kg. '.' is ok to use.");
+                Console.WriteLine($"Your converted float is \"{convertedFloatWeight}\"\n");
+
+                CalculateBmi(convertedFloatWeight, convertedFloatHeight);
+
+                moreEntries = AddAnotherPerson();
+            }
+
+
+        }
+
+        public static bool AddAnotherPerson()
+        {
+            bool moreEntries = true;
+
+            Console.WriteLine("\nWould you like to enter another person? " +
+                                                           "Press Y to enter another or any other button to exit.");
+
+            string checkAnswer = Console.ReadLine().ToLowerInvariant();
+
+            if (checkAnswer != "y")
+            {
+                Console.WriteLine("You chose not to enter another user. Thank you, goodbye.");
+                moreEntries = false;
+            }
+
+            return moreEntries;
+
+        }
+
+
+        public static float CalculateBmi(float weightInKg, float height)
+        {
+            float bmi = weightInKg / (height * height);
+
+            if((weightInKg <= 0) && (height <= 0))
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weightInKg}.");
+                Console.WriteLine($"Height cannot be less than zero, but was {height}.");
+                return -1;
+            }
+
+            if(weightInKg <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Weight cannot be equal or less than zero, but was {weightInKg}.");
+                return -1;
+            }
+
+            if(height <= 0)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine($"Height cannot be equal or less than zero, but was {height}.");
+                return -1;
+            }
+
+            if(weightInKg >= height)
+            {
+                Console.WriteLine("Failed calculating BMI. Reason:");
+                Console.WriteLine("Weight cannot be more or equal to height.");
+                Console.WriteLine($" Height= {height}, Weight= {weightInKg}.");
+                return -1;
+            }
+
+           // Console.WriteLine($"\nYour BMI based on your heigh of {height}m and your weight of {weightInKg}Kg " +
+                          //   $"is {bmi} kg/(m*m)");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4871920)